sdk/schedules/schedule.ipynb

```diff
@@ -0,0 +1,368 @@
+    "    )\n",
+    "    score_with_sample_data.outputs.score_output.mode = \"upload\"\n",
+    "\n",
+    "    eval_with_sample_data = eval_model(\n",
+    "        scoring_result=score_with_sample_data.outputs.score_output\n",
+    "    )\n",
+    "\n",
+    "    # Return: pipeline outputs\n",
+    "    return {\n",
+    "        \"trained_model\": train_with_sample_data.outputs.model_output,\n",
+    "        \"scored_data\": score_with_sample_data.outputs.score_output,\n",
+    "        \"evaluation_report\": eval_with_sample_data.outputs.eval_output,\n",
+    "    }\n",
+    "\n",
+    "\n",
+    "pipeline_job = pipeline_with_components_from_yaml(\n",
+    "    training_input=Input(type=\"uri_folder\", path=parent_dir + \"/data/\"),\n",
+    "    test_input=Input(type=\"uri_folder\", path=parent_dir + \"/data/\"),\n",
+    "    training_max_epochs=20,\n",
+    "    training_learning_rate=1.8,\n",
+    "    learning_rate_schedule=\"time-based\",\n",
+    ")\n",
+    "\n",
+    "# set pipeline level compute\n",
+    "pipeline_job.settings.default_compute = \"cpu-cluster\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1 Create schedule\n",
+    "## 2.1.1 Create schedule with with recurrence pattern"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "create_schedule_recurrence"
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import JobSchedule,ScheduleCronTrigger,ScheduleRecurrenceTrigger # put it to upper once SDK work is ready\n",
+    "from datetime import datetime\n",
+    "from dateutil import tz\n",
+    "from azure.ai.ml.constants import TimeZone, RecurrencePattern\n",
+    "\n",
+    "schedule_name=\"simple_sdk_create_schedule_recurrence\"\n",
+    "\n",
+    "schedule_start_time = datetime.now(tz=tz.tzutc())\n",
+    "recurrence_trigger = ScheduleRecurrenceTrigger(\n",
+    "    frequency=\"day\",\n",
+    "    interval=1,\n",
+    "    schedule=RecurrencePattern(hours=10, minutes=[0, 1]),\n",
+    "    start_time=schedule_start_time,\n",
+    "    time_zone=TimeZone.UTC,\n",
+    "    create_job=pipeline_job\n",
+    ")\n",
+    "\n",
+    "job_schedule = JobSchedule(\n",
+    "    name=schedule_name,\n",
+    "    trigger=recurrence_trigger,\n",
+    "    create_job=pipeline_job\n",
+    ")\n",
+    "\n",
+    "\n",
+    "# ml_client.schedules.begin_create_or_update(schedule=job_schedule)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.1.2 Create schedule with with cron expression"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "create_schedule_cron"
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import JobSchedule,ScheduleCronTrigger,ScheduleRecurrenceTrigger # put it to upper once SDK work is ready\n",
```

Comment:
we should put all imports in first code scetion

Reply:
make sense, updated